sysconfig/cloudinit.go

```diff
@@ -221,7 +223,12 @@ func CloudInitStatus() (CloudInitState, error) {
 		return CloudInitDisabledPermanently, nil
 	}
 
-	out, err := exec.Command("cloud-init", "status").CombinedOutput()
+	lp, err := exec.LookPath("cloud-init")
```

Comment:
I keep thinking this is short for "launchpad", so could you rename this to something like
```suggestion
	ciBinary, err := exec.LookPath("cloud-init")
```